kv/kvserver: TestStoreMetrics failed

kv/kvserver.TestStoreMetrics [failed](https://mesolite.cluster.engflow.com/invocations/default/0803fe61-c0d4-4452-8c21-a4f38a292dfd?testReportRun=1&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on master @ [2a728aa6181095db44cd0264df080d3405098a6f](https://github.com/cockroachdb/cockroach/commits/2a728aa6181095db44cd0264df080d3405098a6f):


```
=== RUN   TestStoreMetrics
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestStoreMetrics1776588172
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:686: DRPC is enabled (via override by TestingGlobalDRPCOption)
    client_metrics_test.go:386: counter rocksdb.block.cache.hits = 0 < min 10
    client_metrics_test.go:386: counter rocksdb.block.cache.hits = 0 < min 10
    client_metrics_test.go:390: -- test log scope end --
test logs left over in: outputs.zip/logTestStoreMetrics1776588172
--- FAIL: TestStoreMetrics (20.93s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStoreMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-61013

kv/kvserver.TestStoreMetrics [failed](https://mesolite.cluster.engflow.com/invocations/default/23deb392-a00b-4eb7-9dea-76826afbc31d?testReportRun=9&testReportShard=30&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on master @ [4559deed7a69713aa0874bd436e5453786517102](https://github.com/cockroachdb/cockroach/commits/4559deed7a69713aa0874bd436e5453786517102):


```
=== RUN   TestStoreMetrics
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestStoreMetrics4155920331
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:146: using test tenant configuration from testserver factory defaults
    test_server_shim.go:638: DRPC is enabled (via testserver factory defaults)
    client_metrics_test.go:386: counter rocksdb.block.cache.hits = 0 < min 10
    client_metrics_test.go:386: counter rocksdb.block.cache.hits = 0 < min 10
    client_metrics_test.go:390: -- test log scope end --
test logs left over in: outputs.zip/logTestStoreMetrics4155920331
--- FAIL: TestStoreMetrics (20.91s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=9</code>
 - <code>shard=30</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStoreMetrics.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/kv/kvserver/client_metrics_test.go
+++ b/pkg/kv/kvserver/client_metrics_test.go
@@ -259,7 +259,7 @@ func TestStoreMetrics(t *testing.T) {
 			Size: storageconfig.BytesSize(512 << 20 /* 512 MiB */),
 		}
 		stickyServerArgs[i] = base.TestServerArgs{
-			CacheSize:  2 << 20, /* 2 MiB */
+			CacheSize:  4 << 20, /* 4 MiB */
 			StoreSpecs: []base.StoreSpec{spec},
 			Knobs: base.TestingKnobs{
 				Server: &server.TestingKnobs{

